server/test/integration/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1502,6 +1502,20 @@ public void shouldMigrateLdapToAuthConfigAsPartOfMigration91() throws Exception
     }
 
     @Test
+    public void shouldAddIdOnConfigRepoAsPartOfMigration94() throws Exception {
+        String configXml = "<cruise schemaVersion='93'>" +
+                "<config-repos>\n" +
+                "   <config-repo plugin=\"json.config.plugin\">\n" +
```

Comment:
@varshavaradarajan -- do we need to test it with multiple `config-repo` tags?
if yes, how it is different from testing with just one?